json_file.py

```diff
@@ -0,0 +1,71 @@
+import json
+import os
+from flask import Response
+
+
+def create_json_file(path: str, output_folder: str,  data: any) -> None:
+    """
+    Create json file
+
+    :param output_folder: The folder for file
+    :param path: The path of the file to be created
+    :param data: Data for a json file
+    :return:
+    """
+    output_path = crate_path_to_json(path, output_folder)
+    try:
+        with open(output_path, 'w') as json_file:
+            json.dump(data, json_file, indent=4)
+    except Exception as e:
```

Comment:
"
Always catch the most specific exception -
https://stackoverflow.com/questions/730764/how-to-properly-ignore-exceptions
"